.github/workflows/update-dependencies.yml

```diff
@@ -51,6 +51,16 @@ jobs:
           npm install
           cd ..
 
+      - name: Login as the automation app
+        # This Action generates a token from the GitHub App and provides it as
+        # an output. It _does_ register that token as a secret so that it will be
+        # filtered from log output automatically
+        id: generate-token
+        uses: tibdex/github-app-token@c95b1c441a4dacd6d24f231b9697a5f9a04c607e
```

Comment:
I elected to use a specific commit hash since we are passing the app id and private key to this Action. But we're also passing the generated token to another action that doesn't use a specific hash, so I am happy to change this to just be `@v1` instead and to track the latest version.